examples/remarketing/upload_store_sales_transactions.rb

```diff
@@ -267,6 +276,19 @@ def build_offline_user_data_job_operations(
       t.transaction_date_time = "2020-05-01 23:52:12"
       t.custom_value = custom_value unless custom_value.nil?
     end
+    if !ad_user_data_consent.nil? || !ad_personalization_consent.nil?
+      job.consent = client.resource.consent do |c|
```

Comment:
Could you rename the `job` variable to something more intuitive, like `op` or `operation`? I realize the name was being used already, but it makes it confusing for users because the block is operating on a single operation in the job, not the job itself.

Reply:
Agreed, i only kept it because it was already there but agree it was confusing. It's done now.

Reply:
Thanks!